client/src/components/files/FileCard.vue

```diff
@@ -4,10 +4,18 @@
   <div
     class="card"
     @click="$emit('click', $event, file)"
+    @mouseenter="isHovered = true"
+    @mouseleave="isHovered = false"
   >
     <div
       class="card-option"
-      @click.stop="$emit('menuClick', $event, file)"
+      v-show="isHovered || menuOpened"
+      @click.stop="
```

Comment:
all the following code makes reading the template laborious, should be moved to a `onSomethingClick` function.